xalia/Win32/NonclientScrollProvider.cs

```diff
@@ -272,12 +345,69 @@ private void SbiChanged(int which)
         public void ParentLocationChanged()
         {
             SbiChanged(SBI_LOCATION);
-            SbiChanged(SBI_STATE); // Sometimes resizing enables/disables scrollbar with no state changed event
         }

```

Comment:
Is `EVENT_OBJECT_VALUECHANGE` expected to be fired every time `EVENT_OBJECT_LOCATIONCHANGE` is?

Reply:
No, but it seems to always fire when the range changes, which seems to be the important case for EVENT_OBJECT_LOCATIONCHAnGE.

Reply:
Got it. Just wanted to make sure removing this was done purposely and not by accident.